src/main/java/com/epam/test_generator/config/WebConfig.java

```diff
@@ -25,6 +23,24 @@ public static PropertySourcesPlaceholderConfigurer propertyConfigInDev() {
         return new PropertySourcesPlaceholderConfigurer();
     }
 
+    @Bean
+    public JavaMailSenderImpl getJavaMailSender() {
+        JavaMailSenderImpl mailSender = new JavaMailSenderImpl();
+        mailSender.setHost("smtp.gmail.com");
+        mailSender.setPort(587);
+
+        mailSender.setUsername("cucumbervarificator@gmail.com");
```

Comment:
How are you going to change this email if google ban it?

Reply:
in future we will use Epam account.
Google is for testing